azext_edge/edge/_help.py

```diff
@@ -1058,211 +1013,108 @@ def load_iotops_help():
 
     helps[
         "iot ops asset endpoint create"
+    ] = """
+        type: group
+        short-summary: Create asset endpoint profiles.
+    """
+
+    helps[
+        "iot ops asset endpoint create opcua"
     ] = """
         type: command
-        short-summary: Create an asset endpoint.
+        short-summary: Create an asset endpoint profile with an OPCUA connector.
         long-summary: |
-                      Either custom location or cluster name must be provided. This command will check
-                      for the existance of the associated custom location and cluster and ensure that
-                      both are set up correctly with the microsoft.deviceregistry.assets extension.
-
                       Azure IoT OPC UA Broker (preview) uses the same client certificate for all secure
                       channels between itself and the OPC UA servers that it connects to.
+
+                      For OPC UA connector arguments, a value of -1 means that parameter will not be used (ex: --session-reconnect-backoff -1 means that no exponential backoff should be used).
+                      A value of 0 means use the fastest practical rate (ex: --default-sampling-int 0 means use the fastest sampling interval possible for the server).
+
+                      For more information on how to create an OPCUA connector, please see aka.ms/opcua_quickstart
```

Comment:
nit, our aka.ms links should use consistent formatting, other links use dash as separator.